addons/slack/form.html

```diff
@@ -134,7 +134,7 @@
                       <Grid container spacing={2}>
                           <Grid item xs={12}>
                               <Grid container spacing={2}>
-                                  {getInput('slack_channel_name', 'Slack channel name', 'The Slack channel to send the notifications to')}
+                                  {getInput('slack_channel_name', 'Slack channel name', 'Please omit the "#" symbol in the channel name')}
```

Comment:
Strip any '#' characters from the input, because you can guarantee people will screw that up.

Reply:
Won't fix. It'll take 30 min of futzing with React, figured it's not worth it for now.